src/Interval.php

```diff
@@ -25,6 +25,8 @@ final class Interval implements JsonSerializable
     private Instant $end;
 
     /**
+     * @deprecated Use {@see Interval::of()} instead, for the consistency with other classes in this library.
```

Comment:
Just:

```
@deprecated Use {@see Interval::of()} instead.
```

please!

Reply:
okay, let's not disclose our intents to the potential enemy :D